scripts/removeWorkspaceDependencies.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+packages_to_remove=(
+  @budibase/backend-core
+  @budibase/bbui
+  @budibase/builder
+  @budibase/cli
+  @budibase/client
+  @budibase/frontend-core
+  @budibase/pro
+  @budibase/sdk
+  @budibase/server
+  @budibase/shared-core
+  # We cannot remove string-templates yet because it cannot be bundled by esbuild as a dependency
+  @budibase/string-templates
+  @budibase/types
+  @budibase/worker
+)
+
+
+root_package_json=$(cat "package.json")
+
+process_package() {
+  local pkg="$1"
```

Comment:
How does this work with loading the dependencies of the packages themselves? Like if we remove `@budibase/backend-core` from the `package.json` how do we fetch its dependencies?

I may be mis-understanding this as well - just trying to understand the script!

Reply:
Ok cool - good to understand!